tests/common/MonoTouch.Dialog/MonoTouch.Dialog.csproj

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)-ios</TargetFramework>
+  </PropertyGroup>
+</Project>
```

Comment:
I think MonoTouch.Dialog.iOS.csproj makes this file redundant?

Reply:
Actually, I was considering it the other way around, just waiting to get the tests to pass before deleting the platform specific versions.

Reply:
Ok, not going to work, reverted